lib/galaxy/tools/parameters/__init__.py

```diff
@@ -2,9 +2,9 @@
 Classes encapsulating Galaxy tool parameters.
 """
 
-from basic import *
-from grouping import *
-from galaxy.util.json import *
+from .basic import DataCollectionToolParameter, DataToolParameter, SelectToolParameter
```

Comment:
You are playing with fire here - which is fine - someone someday needs to fix these :). I would make sure to grep around in odd places such as templates and tools (here and in tools-devteam) for galaxy.tools imports. Sometimes things are imported in parent packages. I think the pattern of importing things from parent modules is fine - as long as these are documented - probably in the form of 

```
__all__ = ['SubMouduleClass', ...]
```

but these things haven't always been documented in this way properly so it is easy to miss stuff.